tom_observations/facilities/lco.py

```diff
@@ -94,11 +113,38 @@ class LCOBaseForm(forms.Form):
     exposure_time = forms.FloatField(min_value=0.1)
     max_airmass = forms.FloatField()
 
+    site = forms.ChoiceField(
+        choices=(('all', 'All Sites'),
+                 ('coj', 'Siding Spring'),
+                 ('cpt', 'Sutherland'),
+                 ('tfn', 'Teide'),
+                 ('tlv', 'Wise'),
+                 ('lsc', 'Cerro Tololo'),
+                 ('elp', 'McDonald'),
+                 ('ogg', 'Haleakala'))
+    )
+
+    imaging_interval = forms.FloatField(
+        label='Interval (hrs). Will schedule exposure count per interval.'
+        )
+
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.fields['proposal'] = forms.ChoiceField(choices=self.proposal_choices())
-        self.fields['filter'] = forms.ChoiceField(choices=self.filter_choices())
-        self.fields['instrument_type'] = forms.ChoiceField(choices=self.instrument_choices())
+        self.fields['proposal'] = forms.ChoiceField(
+            choices=self.proposal_choices()
+            )
+        self.fields['filter'] = forms.ChoiceField(
+            choices=self.filter_choices()
+            )
+        self.fields['instrument_type'] = forms.ChoiceField(
+            choices=self.instrument_choices()
+            )
+
+        self.eph_target = False
+        target = Target.objects.get(pk=kwargs['initial']['target_id'])
+        if target.type == Target.NON_SIDEREAL:
+            if target.scheme == 'EPHEMERIS':
+                self.eph_target = True
 
```

Comment:
Multiple if statements usually are combined into a single if, i.e.:
if target.type == Target.NON_SIDEREAL and  target.scheme == 'EPHEMERIS':
   ...
More a matter of form since both are correct.

Reply:
The reason for the nested statements is because the target.scheme variable is not set unless the target type is == NON_SIDEREAL. So a check of target.scheme == 'EPHEMERIS' alongside target.type == Target.NON_SIDEREAL would crash.